php/ext/google/protobuf/BUILD.bazel

```diff
@@ -0,0 +1,54 @@
+
+# Protobuf PHP extension (PECL)
+load("@rules_pkg//:mappings.bzl", "pkg_files", "pkg_filegroup", "strip_prefix")
+
+filegroup(
+  name = "template_package_xml",
+  srcs = ["template_package.xml"],
+)
+
+pkg_files(
+  name = "utf8_range_files",
+  srcs = [
+    "@utf8_range//:utf8_range_srcs",
+    "@utf8_range//:LICENSE",
+  ],
+)
+
+pkg_files(
+  name = "protobuf_files",
+  srcs = glob([
+    "*.h",
+    "*.c",
+  ]) + [
+    "//:LICENSE",
+    "config.m4",
+    "wkt.inc",
+  ],
+)
+
+pkg_filegroup(
+  name = "release_files",
+  srcs = [
+    ":protobuf_files",
+    ":utf8_range_files",
+  ],
+)
+
+genrule(
+    name = "pecl_dist",
+    srcs = [
+      ":release_files",
+      ":template_package_xml",
+      "//:version_json",
```

Comment:
You should import a version from protobuf_version.bzl instead of using version.json